src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -6591,4 +6591,10 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="WRN_UnreadRecordParameter_Title" xml:space="preserve">
     <value>Parameter is unread. Did you forget to use it to initialize the property with that name?</value>
   </data>
+  <data name="WRN_DoNotCompareFunctionPointers" xml:space="preserve">
+    <value>Function pointer values should not be compared, as JIT tiering and recompilation means that taking the address of a method can return different addresses at different times.</value>
```

Comment:
>should not be compared [](start = 35, length = 22)

Consider saying: "Comparison of function pointers might yield an unexpected result, since pointers to the same function may be distinct." instead.